armi/materials/b4c.py

```diff
@@ -12,8 +12,13 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-"""Boron carbide; a very typical reactor control material."""
-from armi import runLog
```

Comment:
Whoops!

You deleted this import, which is used.

That's why the tests are failing.

Reply:
Oops, this happened when I merged in main. Weird. Fixed!